map/lanelet2_extension/lib/utilities.cpp

```diff
@@ -148,6 +148,7 @@ std::vector<lanelet::BasicPoint3d> resamplePoints(
 
   // Calculate accumulated lengths
   const auto accumulated_lengths = calculateAccumulatedLengths(line_string);
+  if(accumulated_lengths.empty()) return {};
```

Comment:
Function `findNearestIndexPair()` expects at least 2 elements in the `accumulated_lengths`: https://github.com/autowarefoundation/autoware.universe/blob/main/map/lanelet2_extension/lib/utilities.cpp#L121

```suggestion
  if(accumulated_lengths.size() < 2) return {};
```